source/release-notes.txt

```diff
@@ -10,6 +10,39 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
+- Dark mode has been updated with a modern theme. If the modern theme 
+  is enabled in ``Settings>Feature Preview``, Compass defaults to 
+  the modern dark mode theme
+- When Compass detects a potential error importing a CSV file. The  
+  warning banner now shows the value, type, and issue associated with the error (:issue:`COMPASS-6543`)
+- Combine array fields into one in the import CSV preview (:issue:`COMPASS-6766`)
+- Add settings to sidebar menus (:issue:`COMPASS-6796`)
+- Show password only when user focuses on input (:issue:`COMPASS-6161`)
+- Add autocomplete support for $percentile, $median ans $$USER_ROLES (:issue:`COMPASS-6780`) (:issue:`COMPASS-6781`)
+- Toggle state of "Edit connection string" in connection form is now judged also based on the new global preference
+- Update libmongocrypt and driver, re-enable QE tests in 7.0, add test to ensure 6.0 QE data can be read (:issue:`COMPASS-6601`) (:issue:`COMPASS-6602`)
```

Comment:
I'd formulate this similarly to the lines in the mongosh changelog. The exact behavior of the tests isn't relevant, but it **is** very relevant to users that Compass 1.37+ will only be able to do Queryable Encryption with 7.0+ servers (and lower versions of Compass won't work with lower versions of the server), with the exception that pre-7.0 QE data can still be read and decrypted by Compass.

Reply:
Ok I copied the release note from shell to compass:

```
- Compass supports the new Queryable Encryption protocol. Starting
  in v1.37.0, Compass is not compatible with MongoDB server versions
  **earlier than 7.0** when using Queryable Encryption (:issue:`COMPASS-6601`) (:issue:`COMPASS-6602`)

  - When using queryable encryption on pre-7.0 servers, you can decrypt
    encrypted data, but you cannot insert or query data 
  ```

Reply:
Thanks for these updates! This makes a ton more sense.